lib/cucumber/formatter/console.rb

```diff
@@ -80,51 +78,23 @@ def print_element_messages(element_messages, status, kind)
         end
       end
 
-      def print_stats(features, options)
```

Comment:
OK I've updated the labelling on the PR.

I regret that I didn't tease apart the refactoring from the development of the new formatter, but it was late. I guess I still could if you think that would make it easier to review / merge?